rllib/algorithms/dreamerv3/dreamerv3_rl_module.py

```diff
@@ -123,43 +122,6 @@ def get_initial_state(self) -> Dict:
         # Use `DreamerModel`'s `get_initial_state` method.
         return self.dreamer_model.get_initial_state()
 
-    @override(RLModule)
-    def input_specs_inference(self) -> SpecDict:
-        return [Columns.OBS, Columns.STATE_IN, "is_first"]
-
-    @override(RLModule)
-    def output_specs_inference(self) -> SpecDict:
-        return [Columns.ACTIONS, Columns.STATE_OUT]
-
-    @override(RLModule)
-    def input_specs_exploration(self):
-        return self.input_specs_inference()
-
-    @override(RLModule)
-    def output_specs_exploration(self) -> SpecDict:
-        return self.output_specs_inference()
-
-    @override(RLModule)
-    def input_specs_train(self) -> SpecDict:
-        return [Columns.OBS, Columns.ACTIONS, "is_first"]
-
-    @override(RLModule)
-    def output_specs_train(self) -> SpecDict:
-        return [
```

Comment:
Can we leave these as a kind of comment/docstring in the `forward` method? Maybe we do this for all default algos such that we, new RLlib team members and users always know what is returned (received).